spec/deaf_grandma_spec.rb

```diff
@@ -7,12 +7,16 @@
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
     expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
   end
-  
+
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    script.speak("BYE")
+    script.speak("BYE")
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
+    script.speak("BYE")
+    expect(script.speak("HI GRANDMA")).to raise_error SystemExit
```

Comment:
I don't think this passes. Check out the documentation for it:

https://relishapp.com/rspec/rspec-expectations/v/3-7/docs/built-in-matchers/raise-error-matcher